compliance/model/src/test/java/org/eclipse/rdf4j/model/util/IsomorphicBenchmark.java

```diff
@@ -0,0 +1,217 @@
+import org.eclipse.rdf4j.rio.Rio;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@State(Scope.Benchmark)
+@Warmup(iterations = 20)
+@BenchmarkMode({ Mode.AverageTime })
+@Fork(value = 1, jvmArgs = { "-Xms1G", "-Xmx1G", "-XX:+UseSerialGC" })
+//@Fork(value = 1, jvmArgs = {"-Xms8G", "-Xmx8G", "-XX:+UseSerialGC", "-XX:+UnlockCommercialFeatures", "-XX:StartFlightRecording=delay=5s,duration=120s,filename=recording.jfr,settings=profile", "-XX:FlightRecorderOptions=samplethreads=true,stackdepth=1024", "-XX:+UnlockDiagnosticVMOptions", "-XX:+DebugNonSafepoints"})
+@Measurement(iterations = 10)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+public class IsomorphicBenchmark {
+
+	private Model empty = getModel("empty.ttl");
+	private Model blankNodes = getModel("blankNodes.ttl");
+	private Model shacl = getModel("shacl.ttl");
+	private Model longChain = getModel("longChain.ttl");
+	private Model sparqlTestCase = getModel("sparqlTestCase.ttl");
+	private Model spinFullForwardchained = getModel("spin-full-forwardchained.ttl");
+	private Model bsbm = getModel("bsbm-100.ttl");
+	private Model bsbmChanged = getModel("bsbm-100-changed.ttl");
+	private List<Statement> bsbm_arraylist = new ArrayList<>(bsbm);
+	private Model bsbmTree = new TreeModel(bsbm);
+	private Model list = getModel("list.ttl");
+	private Model internallyIsomorphic = getModel("internallyIsomorphic.ttl");
+	private Model manyProperties = getModel("manyProperties.ttl");
+	private Model manyProperties2 = getModel("manyProperties2.ttl");
+
+	private Model empty_2 = getModel("empty.ttl");
+	private Model blankNodes_2 = getModel("blankNodes.ttl");
+	private Model shacl_2 = getModel("shacl.ttl");
+	private Model longChain_2 = getModel("longChain.ttl");
+	private Model sparqlTestCase_2 = getModel("sparqlTestCase.ttl");
+	private Model spinFullForwardchained_2 = getModel("spin-full-forwardchained.ttl");
+	private Model bsbm_2 = getModel("bsbm-100.ttl");
+	private List<Statement> bsbm_arraylist_2 = new ArrayList<>(bsbm);
+	private Model bsbmTree_2 = new TreeModel(bsbm);
+	private Model list_2 = getModel("list.ttl");
+	private Model internallyIsomorphic_2 = getModel("internallyIsomorphic.ttl");
+	private Model manyProperties_2 = getModel("manyProperties.ttl");
+	private Model manyProperties2_2 = getModel("manyProperties2.ttl");
+
+	@Setup(Level.Iteration)
+	public void after() {
+		System.gc();
+	}
+
```

Comment:
What would generally make benchmarks like these easier to use for those of us who don't use IntelliJ, is if they had a main method, like so:


```
    public static void main(String[] args) throws RunnerException {

        Options opt = new OptionsBuilder()
                .include(IsomorphicBenchmark.class.getSimpleName())
                .forks(1)
                .build();

        new Runner(opt).run();
    }
```

If this is present, running these benchmarks directly from Eclipse becomes a lot easier.